feat(l10n): Added translation to Telugu (te)

SpaciousCoder78 opened the pull request:
Just noticed that you published a new app so here's Telugu (te) translation for your new app. 

Hope you have a good day

Jeffser commented:
btw @SpaciousCoder78 I just updated the translations and there are a couple of new strings, could you update them so I can release the next version with Telugu complete? thank you!

SpaciousCoder78 commented:
Yeah I’m on my PC rn I’ll do it and send it in an hour

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Jeffry Samuel ***@***.***>
Sent: Sunday, June 8, 2025 11:13:33 AM
To: Jeffser/Pigment ***@***.***>
Cc: Aryan Karamtoth ***@***.***>; Mention ***@***.***>
Subject: Re: [Jeffser/Pigment] feat(l10n): Added translation to Telugu (te) (PR #2)

[https://avatars.githubusercontent.com/u/69224322?s=20&v=4]Jeffser left a comment (Jeffser/Pigment#2)<https://github.com/Jeffser/Pigment/pull/2#issuecomment-2953581895>

btw @SpaciousCoder78<https://github.com/SpaciousCoder78> I just updated the translations and there are a couple of new strings, could you update them so I can release the next version with Telugu complete? thank you!

—
Reply to this email directly, view it on GitHub<https://github.com/Jeffser/Pigment/pull/2#issuecomment-2953581895>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AVGN6UXZKDWJSCUUWW35GID3CPEQLAVCNFSM6AAAAAB62RFKISVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSNJTGU4DCOBZGU>.
You are receiving this because you were mentioned.Message ID: ***@***.***>
